api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -356,4 +356,10 @@ private void validateInstitution(Institution institutionRequest) {
               });
     }
   }
+
+  public Optional<Institution> getFirstMatchingInstitution(final String contactEmail) {
+    return getInstitutions().stream()
+        .filter(institution -> validateInstitutionalEmail(institution, contactEmail))
```

Comment:
nit: I prefer very small names for lambda arguments, especially if they go out of scope in the same line.

Reply:
If we had a clear style rule one way or another, I'd follow it! For now I'm sticking with my general aesthetic, which is longer-but-clearer parameter names.